plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -478,14 +478,14 @@ CacheStats getStatisticsCacheStats()
     private static final class ColumnsCacheKey
     {
         private final JdbcIdentity identity;
-        private final SchemaTableName table;
+        private final JdbcTableHandle tableHandle;
```

Comment:
@kokosing could you please doublecheck if we really want to generate cache key based on each of the fields present in `equals`/`hashCode` of `JdbcTableHandle`:
`relationHandle`, `constraint`, `sortOrder`, `limit`, `columns`, `nextSyntheticColumnId`. 
What if any of these fields is changed?
Are these fields rather constant in relation to `CachingJdbcClient#metadataCachingTtl` ?